packages/cli/src/util/socket.ts

```diff
@@ -0,0 +1,40 @@
+import chalk from "chalk";
+import { prefix } from "./prefix.js";
+import { WebSocket, WebSocketServer } from "ws";
+
+class WebSocketManager {
+  private static instance: WebSocketManager;
+  private ws: WebSocketServer | null = null;
+
+  private constructor() {
+    this.ws = new WebSocketServer({ port: 3613 });
+
+    this.ws.on("connection", (ws) => {
+      console.log(prefix, chalk.greenBright("connected to extension"));
+      ws.on("error", console.error);
+      ws.on("message", (m) => {
+        if (m.toString() === "disconnecting")
+          console.log(prefix, chalk.redBright("extension disconnected"));
+        if (m.toString() === "received")
+          console.log(prefix, chalk.greenBright.dim("presence updated in extension"));
```

Comment:
Shouldn't the start of a String be Uppercase?

Reply:
No, all logs are lowercase